extensions/cornerstone/src/init.tsx

```diff
@@ -426,10 +426,10 @@ function _showCPURenderingModal(uiModalService, hangingProtocolService) {
     }
   };
 
-  const { unsubscribe } = hangingProtocolService.subscribe(
-    hangingProtocolService.EVENTS.HANGING_PROTOCOL_APPLIED_FOR_VIEWPORT,
-    ({ progress }) => {
-      const done = callback(progress);
+  const { unsubscribe } = HangingProtocolService.subscribe(
```

Comment:
Shouldn't this be small 'h' for the hanging protocol service (i.e. `HangingProtocolService`)? Perhaps I am missing something?